<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>draw_screen</class>
 <widget class="QMainWindow" name="draw_screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: #003049;
}</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{background-color: rgb(234, 226, 183)}</string>
   </property>
   <widget class="QPushButton" name="clear">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>270</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(0, 48, 73);
	color: rgb(247, 127, 0);
	border-radius: 16;
}
QPushButton::disabled{
	background-color: #d62828;
	color: #fcbf49;
	border-radius: 16;
}
QPushButton::hover{
	background-color: #002334;
	color: #fcbf49;
	border-radius: 16;
}

</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>eraser.png</normaloff>eraser.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="image">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>448</width>
      <height>448</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{background-color: white}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>400</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
    border: 4px solid #003049;
    border-radius: 17px;
	background-color:rgb(247, 127, 0);
}
QProgressBar::chunk {
    background-color: #d62828;
	border-radius: 14;
}
</string>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="timer_text">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>400</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>32</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color:#003049
}</string>
    </property>
    <property name="text">
     <string>45</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="start_timer">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>150</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(0, 48, 73);
	color: rgb(247, 127, 0);
	border-radius: 16;
}
QPushButton::disabled{
	background-color: #d62828;
	color: #fcbf49;
	border-radius: 16;
}
QPushButton::hover{
	background-color: #002334;
	color: #fcbf49;
	border-radius: 16;
}

</string>
    </property>
    <property name="text">
     <string>Start Timer</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>play.png</normaloff>play.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>580</y>
      <width>1080</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: #003049;
	color: #f77f00
}</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="next">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>150</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(0, 48, 73);
	color: rgb(247, 127, 0);
	border-radius: 16;
}
QPushButton::disabled{
	background-color: #d62828;
	color: #fcbf49;
	border-radius: 16;
}
QPushButton::hover{
	background-color: #002334;
	color: #fcbf49;
	border-radius: 16;
}

</string>
    </property>
    <property name="text">
     <string>Next Step</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>next.png</normaloff>next.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>270</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(0, 48, 73);
	color: rgb(247, 127, 0);
	border-radius: 16;
}
QPushButton::disabled{
	background-color: #d62828;
	color: #fcbf49;
	border-radius: 16;
}
QPushButton::hover{
	background-color: #002334;
	color: #fcbf49;
	border-radius: 16;
}

</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>loop.png</normaloff>loop.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="instruction">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>441</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: white;
}</string>
    </property>
    <property name="text">
     <string>Start the timer to draw</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>1081</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color:#003049;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>471</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: #f77f00;
	border-radius: 6;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>144</width>
      <height>60</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: #003049;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>tucolorized.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_3</zorder>
   <zorder>clear</zorder>
   <zorder>image</zorder>
   <zorder>progressBar</zorder>
   <zorder>timer_text</zorder>
   <zorder>start_timer</zorder>
   <zorder>label</zorder>
   <zorder>next</zorder>
   <zorder>reset</zorder>
   <zorder>instruction</zorder>
   <zorder>label_2</zorder>
   <zorder>label_4</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>reset</tabstop>
  <tabstop>start_timer</tabstop>
  <tabstop>clear</tabstop>
  <tabstop>next</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
